crates/api/src/comment.rs

```diff
@@ -48,15 +48,15 @@ impl Perform for CreateComment {
     websocket_id: Option<ConnectionId>,
   ) -> Result<CommentResponse, LemmyError> {
     let data: &CreateComment = &self;
-    let user = get_user_from_jwt(&data.auth, context.pool()).await?;
+    let local_user_view = get_local_user_view_from_jwt(&data.auth, context.pool()).await?;
```

Comment:
This `get_local_user_view_from_jwt` now gets a view that joins both the local user and person table, since most API actions need both.